### PR TITLE
fix(pages): removed duplicate 'create proposal' buttons

### PR DESCRIPTION
**Pull request by dmytro-almighty**

### CODE REVIEW

File: types/context.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { Proposal } from 'types/proposal';
+import { DAO } from 'types/dao';
+
+export type UserPermissions = {
+  isCanCreateProposals: boolean;
+};
+
+export type DaoContext = {
```
Comment (AlexeyPonomarenko) [inline types/context.ts:8]: `DaoContext` looks a bit confusing, cause I would actually expect react context for it, but we are not using context. Maybe, `DaoInfo` or `DaoWithPermission` or any other proper name?

File: types/context.ts [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import { Proposal } from 'types/proposal';
+import { DAO } from 'types/dao';
+
+export type UserPermissions = {
+  isCanCreateProposals: boolean;
```
Comment (AlexeyPonomarenko) [inline types/context.ts:5]: `isCanCreateProposals` looks a bit weird.  maybe just `canCreateProposal`?

File: pages/dao/[dao]/treasury/tokens/index.ts [hunk 3]
```diff
@@ -2,35 +2,35 @@ import Tokens, {
...
+  req,
   query,
 }) => {
   const daoId = query.dao as string;
 
-  const [dao, receipts, policyAffectsProposals] = await Promise.all([
-    SputnikHttpService.getDaoById(daoId),
+  CookieService.initServerSideCookies(req?.headers.cookie || null);
+
+  const account = CookieService.get<string | undefined>(ACCOUNT_COOKIE);
```
Comment (AlexeyPonomarenko) [inline pages/dao/[dao]/treasury/tokens/index.ts:17]: Looks like this piece of code is being duplicated. Maybe, there is sense to move it to a separate method and reuse?

File: pages/dao/[dao]/tasks/bounties/BountiesPage.tsx [hunk 4]
```diff
@@ -114,57 +115,58 @@ const BountiesPage: FC<BountiesPageProps> = ({
...
-              size="small"
-              onClick={handleClick()}
-              disabled={!!policyAffectsProposals.length}
+      {bounties.length === 0 ? (
+        <NoResultsView title="No bounties available" />
+      ) : (
+        <>
+          <HeaderWithFilter
+            title={
+              <>
```
Comment (AlexeyPonomarenko) [inline pages/dao/[dao]/tasks/bounties/BountiesPage.tsx:124]: Looks like `<></>` is redundant here and can be removed.

File: types/context.ts (hunk 1 above)
Comment (dmytro-almighty) [reply types/context.ts:8]: But context is not only about react :) its some general information about the Dao

File: pages/dao/[dao]/treasury/tokens/index.ts (hunk 3 above)
Comment (dmytro-almighty) [reply pages/dao/[dao]/treasury/tokens/index.ts:17]: yes, in next iteration i want to extract entire page init method into separate entity

**Review by AlexeyPonomarenko — APPROVED** [review]

**Pull request merged**